example/fail.js

```diff
@@ -3,7 +3,7 @@
 // npm install gulp gulp-eslint
 
 const gulp = require('gulp');
-const gulpUtil = require('gulp-util');
+const log = require('fancy-log');
```

Comment:
Can you rename this to `fancyLog`? I prefer explicitness.

Reply:
Done ;)